spec/models/service_spec.rb

```diff
@@ -13,4 +13,62 @@
   it { is_expected.to allow_value(nil).for(:tier) }
 
   it { is_expected.to have_many(:service_interactions) }
+
+  describe '#provided_by?' do
+    let(:district) { FactoryGirl.build(:local_authority, tier: 'district') }
+    let(:county) { FactoryGirl.build(:local_authority, tier: 'county') }
+    let(:unitary) { FactoryGirl.build(:local_authority, tier: 'unitary') }
+
+    context 'when the tier is blank' do
+      subject { FactoryGirl.build(:service, tier: nil) }
+      it 'is false for a district council' do
+        expect(subject).not_to be_provided_by(district)
```

Comment:
I hadn't noticed this before... is `be_provided_by` a wrapper for `provided_by?(authority)`? Does Rspec generate this automatically?